.NET/Microsoft.Recognizers.Text.NumberWithUnit/English/Extractors/EnglishNumberWithUnitExtractorConfiguration.cs

```diff
@@ -49,7 +52,7 @@ protected EnglishNumberWithUnitExtractorConfiguration(CultureInfo ci)
 
         public Regex NonUnitRegex => NonUnitsRegex;
 
-        public virtual Regex AmbiguousUnitNumberMultiplierRegex => null;
+        public virtual Regex AmbiguousUnitNumberMultiplierRegex => AmbiguousFractionRegex;
 

```

Comment:
I think we can rename this attribute to AmbiguousUnitNumbersRegex and it's value should come from a method that combines AmbiguousUnitNumberMultiplierRegex and AmbiguousFractionRegex.